include/openssl/core_names.h

```diff
@@ -350,6 +349,19 @@ extern "C" {
 #define OSSL_GEN_PARAM_POTENTIAL            "potential" /* integer */
 #define OSSL_GEN_PARAM_ITERATION            "iteration" /* integer */
 
+/* ACVP Test parameters : These should not be used normally */
```

Comment:
Is this intended as public interface, and therefore should they be documented?

Reply:
Hmm that is a tricky question...
They are public really just for the labs validation testing, so they should not be used by 99% of users.

Reply:
I tend to think it should still be documented. Some users may engage other labs, and those labs will need to understand this interface.

Reply:
Ok.